brave/brave-core
# Feat update post onboarding wallet backup UI

Resolves https://github.com/brave/brave-browser/issues/26399
Resolves https://github.com/brave/brave-browser/issues/21524


## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

1. create a wallet (skip backing up)
2. load portfolio page
3. click the backup now prompt
4. enter the wallet password
5. the wallet backup UX should match the UX from onboarding

https://user-images.githubusercontent.com/30185185/199121385-8ad8acfc-eca7-407d-8d41-ffb90c0d462d.mov




https://user-images.githubusercontent.com/30185185/199121365-fdcdeaa0-ecaf-4fbf-9ae8-d7d93e63055d.mov
